### ISSUE TITLE
Error in remedy: putting flowers in mortar/backpack

### ISSUE THREAD

**Issue by rpherbig** [rpherbig/dr-scripts#748]
```
[remedy]>get remed book
You get a book of apprentice remedy instructions from inside your backpack.
>
[remedy]>turn my book to chapter 2
You turn your book to chapter 2, entitled "Simple Remedies".
>
[remedy]>read my book
              -=    Chapter 2 : Simple Remedies    =-

 Page 1: some blister cream                      Page 4: some wart salve
 Page 2: some itch salve                         Page 5: some stomach tonic
 Page 3: some moisturizing ointment              
>
[remedy]>turn my book to page 1
You turn your book to page 1, instructions for crafting some blister cream.
>
[remedy]>study my book
You scan the blister cream instructions with a glance and comprehend all but several minor details in the text.
You now feel ready to begin the crafting process.
Roundtime: 6 sec.
>
[remedy]>put my book in my backpack
You put your book in your backpack.
>
[remedy]>get my mortar
You get an iron mortar from inside your backpack.
>
[remedy]>get my red flowers
You get some dried red flowers from inside your backpack.
>
[remedy]>put red flowers in my mortar
You vigorously rub the flowers alongside the mortar to scrape some shavings into the mixture.
>
[remedy]>put my red flowers in my backpack
>
What were you referring to?
>
[remedy]>get my pestle
You need a free hand to pick that up.
>
[remedy: *** No match was found after 15 seconds, dumping info]
[remedy: messages seen length: 2]
[remedy: message: You need a free hand to pick that up.]
[remedy: message:  Khri Dampen  (Fading)]
[remedy: checked against [/^You get/i, /^You are already/i, /^What do you/i, /^What were you/i, /You pick up/i, /can't quite lift it/i]]
[remedy]>crush my red flowers with my pestle
You need to be holding the pestle first.
>
[remedy]>put my flowers in my backpack
You put your flowers in your backpack.
```

**Comment by Dartellum** [comment]
If the question is about:
[remedy]>put my red flowers in my backpack
>
What were you referring to?

That is correct if you had combined the herbs to a stack of more than 25. The mortar can only hold 25 flowers. I would need to think how to do this better and seek advice.

**Comment by rpherbig** [comment]
Should be fixed in #837
